src/main/webapp/data/userMapData.json

```diff
@@ -67,6 +70,7 @@
       "University of South Australia",
       "University of Sydney",
       "University of Tasmania",
+      "University of Technology Sydney",
```

Comment:
Repeated twice

Reply:
Hi @shaharyar-shamshi  , I'll change it.